### ISSUE TITLE
Generate table of contents under "Options" heading for all docs

### ISSUE THREAD

**Issue by Murderlon** [transloadit/uppy.io#45]
The sidebar with links for every docs page is convenient to quickly navigate:

<img width="226" alt="Screenshot 2022-12-23 at 12 46 38" src="https://user-images.githubusercontent.com/9060226/209331118-87e68680-057f-482d-89b2-1a236de03fc0.png">

However it doesn't contain all options. Instead it would be nice to [generate](https://github.com/remarkjs/remark-toc/) a list of links of all the options under the "Options" heading in the page (so not in the sidebar).

**Comment by kvz** [comment]
Would be nice to have post-launch

**Comment by Murderlon** [comment]
This is blocked by https://github.com/facebook/docusaurus/issues/5379
